src/main/java/com/byznass/tiolktrack/postgres/dao/PostgresUserProvider.java

```diff
@@ -0,0 +1,62 @@
+package com.byznass.tiolktrack.postgres.dao;
+
+import com.byznass.tiolktrack.kernel.TiolkTrackException;
+import com.byznass.tiolktrack.kernel.dao.NoUserWithSuchIdException;
+import com.byznass.tiolktrack.kernel.dao.UserProvider;
+import com.byznass.tiolktrack.kernel.model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class PostgresUserProvider implements UserProvider {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(PostgresUserProvider.class);
+
+	private final Connection connection;
+
+	@Inject
+	public PostgresUserProvider(Connection connection) {
+
+		this.connection = connection;
+	}
+
+
+	@Override
+	public User getUser(String userId) throws NoUserWithSuchIdException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.dao.NoUserWithSuchIdException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)